### PR TITLE
Con 2391

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationInverseTransition.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require(['jquery', 'wikia.window'], function($, win) {
+	'use strict';
+	var $globalNav = $('#globalNavigation');
+
+	function changeStateOnScroll() {
+		if (win.pageYOffset === 0 && $globalNav.hasClass('inverse')) {
```
Comment (hakubo) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationInverseTransition.js]: inverse could be a var
2 magic numbers ahead!

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss [hunk 2]
```diff
@@ -12,6 +12,7 @@ $start-a-wikia-logo-size: 30px;
 	color: $nav-foreground-dark-color;
 	height: $global-navigation-height;
 	position: fixed;
+	transition-duration: 0.5s;
```
Comment (hakubo) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss]: .5s

**Comment by hakubo** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Comment by bognix** [comment]
instead of this PR we should merge https://github.com/Wikia/app/pull/6199/files to dev.
Closing this one.

**Pull request closed without merging**